src/Features/OnboardingTasks/TaskLists.php

```diff
@@ -138,6 +138,14 @@ public static function get_lists( $extended_tasks = array() ) {
 		return self::$lists;
 	}
 
+	/**
```

Comment:
Just used for testing currently; perhaps there is a better method?

Reply:
I think this is a reasonable way to handle this.